colossalai/shardformer/policies/bert.py

```diff
@@ -38,11 +39,13 @@ def config_sanity_check(self):
 
     def preprocess(self):
         self.tie_weight = self.tie_weight_check()
+        self.enable_bias_gelu_fused = self.model.config.hidden_act == "gelu"
```

Comment:
`= self.shard_config.enable_jit_fused and self.model.config.hidden_act == "gelu"`

Reply:
updated.